tests/ragger/eip712/input_files/00-simple_mail-data.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
That's right, the whole APDU exchange for an EIP712 message signature is complex and the python Ethereum client only produces valid APDUs.
It would however be possible to send the APDU out of order, but sending straight-up incorrect ones, not possible and I've not found a way so far to do it elegantly.